codejail/jail.py

```diff
@@ -147,7 +148,8 @@ def safe_exec(self, code, globals_dict, files=None, python_path=None, slug=None,
                 files.append(pydir)
 
         stdin = json.dumps([code, json_safe(globals_dict)])
-        jailed_code = self.lang.safe_exec_template % {'python_path': '\n'.join(pypath_code)}
+        jailed_code = self.lang.safe_exec_template % {'python_path': '\n'.join(
+            pypath_code)}
```

Comment:
Please don't change the formatting of lines you aren't changing. There's no reason to wrap this line like this.

Reply:
I thought we are following PEP8 standards.